web/src/compose.js

```diff
@@ -371,7 +371,7 @@ export function update_email(user_id, new_email) {
 
 function insert_video_call_url(url, target_textarea) {
     const link_text = $t({defaultMessage: "Click to join video call"});
```

Comment:
@alya this can be a follow-up, but do we want this sentence to end with a `.`? It's always bugged me a bit that these don't.

Reply:
After chatting with @alya, how about just `[Join video call.]({url})` as the format here.

Reply:
@timabbott Here it is: #25095